open-api.yaml

```diff
@@ -323,9 +323,9 @@ components:
             indexedDocuments:
               type: integer
               description: Number of documents finally indexed for documentAdditionOrUpdate task or a documentAdditionOrUpdate batch of tasks.
-            receivedDocumentsIds:
```

Comment:
`matchedDocuments` is returned for `delete-batch` task, and was chosen because, in the future, we expect to allow the deletion of documents by filters when deleting documents (with `delete-batch` or not)
`receivedDocuments` here are literally the documents sent by the users so received by meilisearch

For me it's consistent. Maybe @gmourier has a different opinion

Reply:
This is only for the `documentDeletion` task type. Since there are multiple ways of deleting documents, by an id, by ids, and in the future by a filter, it seems better to call it `matchedDocuments`

`matchedDocuments` will be less precise in its naming than `receivedDocumentsIds` in the case of deletion by identifier but will be much more straightforward for deletion by a filter.